docs/source/tutorials/getting-started.ipynb

```diff
@@ -0,0 +1,675 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Background\n",
+    "PSI/J (Portable Submission Interface for Jobs), is a Python abstraction layer which allows your HPC application to run virtually anywhere. <br> <br>\n",
+    "![psij-overview-visual](./images/psij-overview.png)\n",
+    "<br>\n",
+    "<br>\n",
+    "PSI/J automatically translates abstract job specifications into concrete scripts and commands to send to the scheduler. PSI/J has a number of advantages:\n",
+    "1. **Runs entirely in user space**: no need to wait for infrequent deployment cycles, it's easy to leverage built-in or community-provided plugins\n",
+    "2. **An asynchronous modern API for job management**: a clean Python API for requesting and managing jobs.\n",
+    "3. **Supports the common batch schedulers**: we test PSI/J across multiple DOE supercomputer centers. It’s easy to test PSI/J on your systems and share the results with the community.\n",
+    "4. **Built by the HPC community, for the HPC community**: PSI/J is based on a number of libraries used by state-of-the-art HPC workflow applications.\n",
+    "5. **PSI/J is an open source project**: we are establishing a community to develop, test, and deploy PSI/J across many HPC facilities."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install from pip ..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "bat"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!bash -c \"pip install psij-python\""
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## ... Or Install from Source"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "bat"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!bash -c \"python3 --version\"\n",
+    "!bash -c \"python3 -m venv psij-tutorial-venv\"\n",
+    "!bash -c \"source ./psij-tutorial-venv/bin/activate\"\n",
+    "!bash -c \"pip install --upgrade pip\"\n",
+    "!bash -c \"pip install git+https://github.com/ExaWorks/psij-python.git\""
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Overview \n",
+    "In PSI/J’s terminology, a [Job](https://exaworks.org/psij-python/docs/.generated/psij.html#psij.job.Job) represents an executable plus a bunch of attributes. Static job attributes such as resource requirements are defined by the [JobSpec]() at creation. Dynamic job attributes such as the [JobState]() are modified by [JobExecutors]() as the [Job]() progresses through its lifecycle.\n",
```

Comment:
The link IS broken. It's supposed to be `.../#docs/...` not `.../docs/..`. So [https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job)

Reply:
After checking again, it seems that opening a link in a new tab (via CMD-Click), or right-clicking and selecting "copy link address" leads to the unwanted behavior of not including the # before "docs". Since this is the case, I'll go ahead and fix & include them within the notebook.

Clicking on the hyperlinks from within the documentation loads them successfully.

My issue where links don't load successfully from within a jupyter notebook (ran via vscode) still persists. For example, I have a hyperlink for `job`, which correctly links to:
https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job

However, when I click it from within vscodes view of the notebook, it translates it to:
https://exaworks.org/psij-python/#docs/.generated/psij.html/%23psij.job.Job

Which breaks the link and gives me a 404.

It may be an issue that I'm editing a notebook over ssh, but the link is trying to be opened on my local machine. I'm not sure but that's my best guess.

Reply:
> After checking again, it seems that opening a link in a new tab (via CMD-Click), or right-clicking and selecting "copy link address" leads to the unwanted behavior of not including the # before "docs". Since this is the case, I'll go ahead and fix & include them within the notebook.

Maybe it's time to revamp how our webpages are done. 

> However, when I click it from within vscodes view of the notebook, it translates it to: https://exaworks.org/psij-python/#docs/.generated/psij.html/%23psij.job.Job
> 
> Which breaks the link and gives me a 404.

Is that how we plan to run the tutorial (i.e., via vscode)?

Reply:
> Is that how we plan to run the tutorial (i.e., via vscode)?

I would think we should use a dedicated docker container for now - https://github.com/ExaWorks/SDK/tree/master/docker/tutorials